src/utils/api-utils/UrlSearchParamBuilder.js

```diff
@@ -10,8 +10,8 @@ export class UrlSearchParamBuilder {
     return this;
   }
 
-  setFilters(...filters) {
-    this.params.append('filter', filters.join('&&'));
+  setFilters(filter) {
```

Comment:
nit: could this be named `setFilter` without the 's'? Then if we ever need to re-introduce the above removed function for multiple separate filters we can with it's appropriate name.

Reply:
Changed to setFilter